snmp_dell/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "Dell",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "snmp_dell",
+  "metric_prefix": "snmp.",
+  "metric_to_check": [
+    "snmp.fanSpeed",
+    "snmp.systemStateChassisStatus",
+    "snmp.cacheDeviceStatus"
+  ],
+  "creates_events": false,
+  "short_description": "Collect SNMP metrics from your Dell network devices.",
```

Comment:
@naaltman and here

Reply:
small nit to remove `network` and just say `Collect metrics from Dell devices` remove the SNMP and word network, since for e.g Isilon is a storage device and not a "network" device, some network engineers get picky about this which is why I mention it